misc/working_w_l10n_agency.md

```diff
@@ -0,0 +1,88 @@
+# Working with a Localization Agency
+
+This document is a guideline for project manager who interacts with a localization agency that is already vetted.
+
+## Make it official
+
+A series of activities need to happen before an l10n project can start.
+
+* A signed master service agreement between Mozilla and the vendor. Person signed on Mozilla side could be the PM or PM’s manager.
+* Account Payable form that contains banking information, billing cycle (monthly, quarterly or as soon as) and payment terms (within 30 or 45 days); billing email address.
+* An official rate sheet that includes the following but not limited to:
+** Word rate for each of the interested locales as in new words, 100% match, fuzzy match for translation, review, proofreading
```

Comment:
Use Atom to edit these documents and check the preview. These secondary items are currently broken.
https://github.com/mozilla-l10n/documentation/blob/982d1d4f0ea030370b2f7cf5e681d3e621d8971f/misc/working_w_l10n_agency.md

End each list item with a period.

As someone who has zero experience in dealing with agencies, I would love a short explanation of what each elements is.